standards/examples/turbogeth.md

```diff
@@ -0,0 +1,74 @@
+## Turbogeth managed service
+
+This article contains examples of Demands & Offers which cover the a Managed Service example (turbogeth).
+
+Compared to Standards v0.6.0, the sample includes a couple of extensions:
+
+### Perpetual Agreement
+
+No `golem.srv.comp.expiration` property specified on the Demand side indicates no expected Agreement end date.
+
+### "Eth" Application Namespace
+
+Any "Ethereum-specific" properties would be maintained in this namespace. This sample includes:
+
+`golem.srv.app.eth.rpc-port` - Indication of RPC endpoint port exposed by the provisioned Geth service.
+
+### Managed Turbogeth runtime
+
+`golem.runtime.name=turbogeth-managed` - Indicated a Managed Turbogeth service to be launched
+
+### PAYU payment scheme enhancements
+
+Two negotiable properties to determine Debit Note and Payment intervals:
+
+`golem.com.scheme.payu.debit-note-interval-sec?` - Debit Notes issue interval
+`golem.com.scheme.payu.payment-interval-sec?` - Expects Payments to be made within specific interval after Debit Note is issued
+
+### Sample Offer
+
+```
+"properties": {
+    "golem.com.payment.platform.erc20-mainnet-glm.address": "0x128282ae0ba38689f215cb3cc278008620f50253",
+    "golem.com.payment.platform.zksync-mainnet-glm.address": "0x128282ae0ba38689f215cb3cc278008620f50253",
+    "golem.com.pricing.model": "linear",
+    "golem.com.pricing.model.linear.coeffs": [
+        0.0002777777777777778,
+        0.001388888888888889,
+        0.0
+    ],
+    "golem.com.scheme": "payu",
+    "golem.com.scheme.payu.debit-note-interval-sec?": 60, // Debit Notes issues every minute
+    "golem.com.scheme.payu.payment-interval-sec?": 3600,  // Expects Payments to be made within 1 hour 
```

Comment:
Did we already decided this? That all DebitNotes have `DueDate` and only Payment Driver will optimize transactions by batching them?

Reply:
...but all this must be negotiated first...

Reply:
My question was related to comment that payments are expected to be made in 1h, what suggests, that each DebitNote has DueDate. But we already decided this, so this comment is outdated.